### ISSUE TITLE
When I evaluate a model in the universal setting, change the image size.

### ISSUE THREAD

**Issue by brian2023707** [cfifty/CAML#17]
Hello, I am evaluating the model using my own dataset, and I am already seeing good performance. However, I would like to try increasing the image size to see if it can improve the accuracy.

Could you please let me know if I can change the image size directly during evaluation, or if it is not possible to change it during evaluation and I need to retrain the model?

**Comment by cfifty** [comment]
Hey -- you'd need to do two things:

1. Use a different pre-trained image encoder to operate on higher-resolution images.
2. Re-trained the in-context learning module (what we call CAML) to operate on the representations produced by the higher-resolution-input image encoder.

**Comment by brian2023707** [comment]
I’ve set all datasets to 6-way, adjusted num_classes to 6, and modified this warning to 6 as well:

if torch.unique(support_labels).shape[0] != 6:
    print(f'Malformed batch with only {torch.unique(support_labels).shape[0]} unique support examples. Skipping.')

During training, I occasionally see "Malformed batch with only 5 unique support examples. Skipping." Is this normal? Could it be because the training set doesn’t support more ways?
